README.md

```diff
@@ -4,9 +4,11 @@
 
 The purpose of this test is to create simple game logic based on the detailed specifications and prove its correctness.
 
-We realise that everyone has different levels of skill and experience when it comes to development, so we have split the test into three sections. If you do not have the knowledge to complete all sections, that's okay. We want to see how you approach the problem and get a feel for how you code. This also gives you an opportunity to present yourself at your best while working in the comfort of your home. 
+We realise that everyone has different levels of skill and experience when it comes to development, so we have split the test into multiple tasks. If you do not have the knowledge to complete all questions, that's okay. We want to see how you approach the problem and get a feel for how you code. This also gives you an opportunity to present yourself at your best while working in the comfort of your home.
```

Comment:
I think this was not a necessary change. If anything, "If you do not have the knowledge" should be "If you are unable to complete".